metal/interrupt.h

```diff
@@ -61,6 +66,15 @@ inline void metal_interrupt_init(struct metal_interrupt *controller)
     return controller->vtable->interrupt_init(controller);
 }
 
+/*!
+ * @brief Return interrupt controller label
+ *
+ * @param controller The handle for the interrupt controller
+ */
+inline char * metal_interrupt_label(struct metal_interrupt *controller)
```

Comment:
This should be more explicit about what it means. Maybe `metal_interrupt_get_controller_compatible_string()`?

Reply:
This API has been replace with metal_interrupt_get_controller()